### ISSUE TITLE
Change requirements for bitsandbytes to 0.37.2

### ISSUE THREAD

**Issue by Nan-Do** [oobabooga/text-generation-webui#2248]
### Describe the bug

There is a bug in the bitsandbytes 0.38.x that increases the memory consumption by a lot when saving a model, please see this https://github.com/TimDettmers/bitsandbytes/issues/324, there are issues opened in relation to this too here too, please see https://github.com/oobabooga/text-generation-webui/issues/1669

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Train a model and try to save it to disk

### Logs

```shell
Not related
```

### System Info

```shell
Not related
```

**Comment by Ph0rk0z** [comment]
I think this is a bit up to the user?

**Comment by Nan-Do** [comment]
Yeah I meant it just a suggestion, feel free to close the issue, but it's just a little change that can make life easier for a lot of people, if you are using that library it's because you want to use it to train models on cards with less memory and that bug pretty much negates that.

**Comment by Ph0rk0z** [comment]
He just released a bunch of updates too. I think a new version is coming soon.
